src/hooks/usePaginatedReportActions.ts

```diff
@@ -11,10 +12,12 @@ function usePaginatedReportActions(reportID?: string, reportActionID?: string) {
     // Use `||` instead of `??` to handle empty string.
     // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
     const reportIDWithDefault = reportID || '-1';
+    const report = ReportUtils.getReport(reportIDWithDefault);
```

Comment:
`useOnyx` is already a part of this file, so I assume it works just fine? Generally, `ReportUtils.getReport()` should be avoided because anything that it returns won't get updated if the onyx data updates.